integration/operate-first/README.md

```diff
@@ -0,0 +1,92 @@
+cd hack/tools
+./install_yq.sh
+./install_helm.sh
+```
+2. Go back to the `integration/operate-first` folder and set up the Python environment there
+```bash
+cd samples/operate-first
+pipenv install
+pipenv shell
+```
+3. Run Makefile to generate new YAML files from the Helm charts:
+```bash
+make all
+```
+
+After the cluster-scoped YAML files are generated, create a PR to the [operate-first/apps](https://github.com/operate-first/apps) repository in the [cluster-scope/base](https://github.com/operate-first/apps/tree/master/cluster-scope/base) directory with the YAML files in the subdirectories organized by resource type. More documentation about contributing to the `operate-first/apps` repository can be found [here](https://github.com/operate-first/apps/tree/master/docs/content)
+
+## Deploying Namespace-Scoped resources
+Operate First has an [ArgoCD](https://argo-cd.readthedocs.io/en/stable/) instance deployed on MOC that can be used to deploy OpenShift resources located on a Git Repository. Fybrik has been onboarded to ArgoCD by following [these instructions](https://github.com/operate-first/apps/blob/master/docs/content/argocd-gitops/onboarding_to_argocd.md) and an ArgoCD project has been created for Fybrik. You can login to [the ArgoCD instance](https://argocd.operate-first.cloud/applications?proj=&sync=&health=&namespace=&cluster=&labels=) with the same login method as above. We have deployed 3 [ArgoCD applications](https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#applications) which are automatically synced with the latest release of Fybrik. See the ArgoCD applications below pointing to the corresponding Fybrik repositories:
+* fybrik -> https://github.com/fybrik/charts
+* arrow-flight-module -> https://github.com/fybrik/arrow-flight-module
+* vault -> https://github.com/fybrik/fybrik
+
+The complete ArgoCD application manifests have been added to the `operate-first/apps` repository [here](https://github.com/operate-first/apps/tree/master/argocd/overlays/moc-infra/applications/envs/moc/smaug/fybrik)
+
+## Running the Fybrik quickstart on Operate First
+1) Follow the steps in [Fybrik notebook sample](https://fybrik.io/v0.5/samples/notebook/) to prepare a dataset to be accessed by the notebook, register the dataset in a data catalog, and define data access policies.
+2) Access the JupyterHub instance deployed on the Smaug OpenShift cluster [here](https://oauth-openshift.apps.smaug.na.operate-first.cloud/oauth/authorize?response_type=code&redirect_uri=https%3A%2F%2Fjupyterhub-opf-jupyterhub.apps.smaug.na.operate-first.cloud%2Fhub%2Foauth_callback&client_id=system%3Aserviceaccount%3Aopf-jupyterhub%3Ajupyterhub-hub&state=eyJzdGF0ZV9pZCI6ICIwY2ZkYzYwMjA4MjY0OGZiYWY5MDk3OWJkOGFhZjE4NyIsICJuZXh0X3VybCI6ICIvaHViLyJ9&scope=user%3Ainfo) and login with the above method. 
+3) Start a notebook server using the Elyra Notebook Image or any image of your choosing
+4) Create a notebook with the following cells:
+```python
```

Comment:
@eletonia I think you can remove the snippet below and also refer to the notebook sample section that runs the notebook. This can prevent inconsistency when things are updated (for example change in the arrow-flight endpoint)

Reply:
Done